content/fr/docs/community/history.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "L'histoire du Projet"
+description: "Fournit un aperçu général de l'historique du projet."
+weight: 4
+---
+
+Helm est un [projet de la CNCF](https://www.cncf.io/projects/), qui a été [récompensé](https://helm.sh/blog/celebrating-helms-cncf-graduation/).
```

Comment:
```suggestion
Helm est un [projet gradué](https://helm.sh/blog/celebrating-helms-cncf-graduation/) [de la CNCF](https://www.cncf.io/projects/).
```

Reply:
In french the word "gradué" is not common and used in very specific them. I recomend to use the word "récompensé" instead.

Reply:
```suggestion
Helm est un [projet récompensé](https://helm.sh/blog/celebrating-helms-cncf-graduation/) par la [CNCF](https://www.cncf.io/projects/).
```